src/components/Footer/Footer.js

```diff
@@ -3,13 +3,17 @@ import "./Footer.css";
 import ppLogo from "../../img/pp-logo.png";
 import ccIcon from "../../img/cc-icon.png";
 import fbIcon from "../../img/pp-social-fb.png";
-import rssIcon from "../../img/pp-social-rss.png";
 import twitterIcon from "../../img/pp-social-tw.png";
 import { injectIntl } from "react-intl";
 import { Link } from "react-router-dom";
+import { ShareButtons } from "react-share";
 
 export class Footer extends React.Component {
   render() {
+    const { FacebookShareButton, TwitterShareButton } = ShareButtons;
+    let currentUrl = "https://participedia.xyz";
```

Comment:
We should probably pull that out of an environment variable, as this will break when we move to the production site.  e.g. CREATE_REACT_ROOT_URL or something.